Makefile

```diff
@@ -27,7 +27,7 @@ SILENT ?= @
 UNIT_TEST_IGNORE := "stackrox/rox/sensor/tests|stackrox/rox/operator/tests|stackrox/rox/central/reports/config/store/postgres|stackrox/rox/central/complianceoperator/v2/scanconfigurations/store/postgres|stackrox/rox/central/auth/store/postgres|stackrox/rox/scanner/e2etests"
```

Comment:
> perhaps the PR should remain split, WDYT?
Yes, I agree.

Reply:
I'll try to go through these files before end-of-week.  My current plan is to test these out by locally tagging using the `.0` strategy and running the tests to see if they break.

Reply:
I've started this task today, but I haven't gotten very far yet.  I'll work on it some more tomorrow.

Reply:
To see if there were any test regressions, I ran `make go-unit-tests` and `make shell-unit-tests` against both `master` and `klape/rox-22250` branches.  I saved the output from `master` under the folders `test-output-master` and `shell-test-output-master`, respectively, and left the output from `klape/rox-22250` in the default locations `test-output` and `shell-test-output`.

This is the comparison of failures:

```
$ rg FAIL: test-output-master/test.log test-output/test.log
test-output-master/test.log
19138:--- FAIL: TestRateLimitedLogger (0.02s)
19147:    --- FAIL: TestRateLimitedLogger/TestRateLimitedFunctionsCacheEviction (0.00s)
19148:    --- FAIL: TestRateLimitedLogger/TestRateLimitedFunctionsDebugLBurst (0.00s)
19149:    --- FAIL: TestRateLimitedLogger/TestRateLimitedFunctionsErrorLBurst (0.00s)
19150:    --- FAIL: TestRateLimitedLogger/TestRateLimitedFunctionsInfoLBurst (0.00s)
19151:    --- FAIL: TestRateLimitedLogger/TestRateLimitedFunctionsSameLimiterDifferentLogs (0.00s)
19152:    --- FAIL: TestRateLimitedLogger/TestRateLimitedFunctionsWarnLBurst (0.00s)

test-output/test.log
19256:--- FAIL: TestRateLimitedLogger (0.02s)
19265:    --- FAIL: TestRateLimitedLogger/TestRateLimitedFunctionsCacheEviction (0.00s)
19266:    --- FAIL: TestRateLimitedLogger/TestRateLimitedFunctionsDebugLBurst (0.00s)
19267:    --- FAIL: TestRateLimitedLogger/TestRateLimitedFunctionsErrorLBurst (0.00s)
19268:    --- FAIL: TestRateLimitedLogger/TestRateLimitedFunctionsInfoLBurst (0.00s)
19269:    --- FAIL: TestRateLimitedLogger/TestRateLimitedFunctionsSameLimiterDifferentLogs (0.00s)
19270:    --- FAIL: TestRateLimitedLogger/TestRateLimitedFunctionsWarnLBurst (0.00s)

$ rg 'failures="[^0]"' shell-test-output-master/report.xml
44:<testsuite name="ci/bats/lib_junit.bats" tests="13" failures="3" errors="0" skipped="0" time="0.707" timestamp="2024-06-13T18:33:58" hostname="klape-thinkpadp1gen4i.remote.csb">
141:<testsuite name="ci/bats/lib_pr_has_label.bats" tests="4" failures="2" errors="0" skipped="0" time="0.182" timestamp="2024-06-13T18:33:58" hostname="klape-thinkpadp1gen4i.remote.csb">
202:<testsuite name="ci/logcheck/check-restart-logs_test.bats" tests="17" failures="3" errors="0" skipped="0" time="5.467" timestamp="2024-06-13T18:34:05" hostname="klape-thinkpadp1gen4i.remote.csb">
306:<testsuite name="ci/store-artifacts.bats" tests="7" failures="4" errors="0" skipped="0" time="0.432" timestamp="2024-06-13T18:34:06" hostname="klape-thinkpadp1gen4i.remote.csb">

$ rg 'failures="[^0]"' shell-test-output/report.xml
44:<testsuite name="ci/bats/lib_junit.bats" tests="13" failures="3" errors="0" skipped="0" time="0.712" timestamp="2024-06-13T18:36:39" hostname="klape-thinkpadp1gen4i.remote.csb">
141:<testsuite name="ci/bats/lib_pr_has_label.bats" tests="4" failures="2" errors="0" skipped="0" time="0.191" timestamp="2024-06-13T18:36:40" hostname="klape-thinkpadp1gen4i.remote.csb">
202:<testsuite name="ci/logcheck/check-restart-logs_test.bats" tests="17" failures="3" errors="0" skipped="0" time="5.442" timestamp="2024-06-13T18:36:47" hostname="klape-thinkpadp1gen4i.remote.csb">
306:<testsuite name="ci/store-artifacts.bats" tests="7" failures="4" errors="0" skipped="0" time="0.438" timestamp="2024-06-13T18:36:47" hostname="klape-thinkpadp1gen4i.remote.csb">
```

So the results look to be unchanged.  Next I'll go through the tests to see if there are any that should be changed because they specifically testing the version string.